types/openid-client/openid-client-tests.ts

```diff
@@ -111,4 +107,8 @@ async (req: IncomingMessage) => {
     await client.revoke('token', 'hint');
     client.revoke('token', 'hint', {});
     client.revoke('token', 'hint', { revokeBody: {}, clientAssertionPayload: {} });
+
+    const res = await client.refresh('token');
```

Comment:
One more small thing. Either use `res` to test `TokenSet`  members and rename the local to `refreshResponse` (to avoid later naming conflicts) or just drop the local.